app/controllers/concerns/exception_handling.rb

```diff
@@ -6,17 +6,11 @@
 module ExceptionHandling
   extend ActiveSupport::Concern
 
-  SKIP_SENTRY_EXCEPTION_TYPES = [
-    Common::Exceptions::Unauthorized,
-    Common::Exceptions::RoutingError,
-    Common::Exceptions::Forbidden,
-    Breakers::OutageException
```

Comment:
I'm not sure if Breakers::OtageException being excluded was getting tested anywhere. Similar to the comments I made below, I think this should be a warn or info, not excluded altogether.